javascript-loops-practice/find-index.js

```diff
@@ -0,0 +1,10 @@
+/* exported findIndex */
+function findIndex(array, value) {
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === value) {
+      return i;
+    } else if (i + 1 === array.length) {
+      return -1;
+    }
```

Comment:
Instead of having this conditional run on each iteration of the loop, you can just `return -1` below the loop, since the entire loop will have run by that point, and if the value wasn't found, you can safely assume that you should return `-1`